packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -320,13 +375,115 @@ export const Webchat = forwardRef((props, ref) => {
         ...webchatState.theme.triggerButtonStyle
       }}
     >
-      <img
+      {triggerImage && (
+        <img
+          style={{
+            height: 50
```

Comment:
Maybe put some style option in order to define the size of the triggerImage. I don't know if it's better this option or force the user to create a CustomTriggerButton